wiki/Welcome/fr.md

```diff
@@ -0,0 +1,41 @@
+[//]: # (Traduit par cth103)
+# Bienvenue
+
+Bienvenue dans _osu!_, un jeu rythmique gratuit développé par peppy avec quatre modes de jeu: osu!standard, un simulateur de cliquage de cercles; osu!taiko, un emulateur de tambour; osu!catch, attraper des fruits; osu!mania, une manie casseur de clé.
```

Comment:
`un jeu rythmique` ? , `un jeu de rythme` me semble plus naturel.
Il manque la notion de `free-to-win` qui est très importante selon moi.
Il manque un accent à `emulateur`.
`attraper des fruits` ne colle pas avec les deux éléments précédents.
`casseur de clé` gné ? je mettrais quelque chose du genre `une manie mâcheuse de touches`. Eh oui, `touches` et pas `clé`